<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ian Gebbie</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linux HP Connect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Wi&amp;ndow</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionClose_all"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionMessages"/>
    <addaction name="actionMonitor"/>
    <addaction name="actionCalculators"/>
    <addaction name="actionContent"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_HP_Connect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QDockWidget" name="dwCalculator">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>184</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="wCalculator">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>150</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTreeView" name="tvCalculators">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCalculators"/>
   <addaction name="actionContent"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_All"/>
   <addaction name="actionCreateNew"/>
   <addaction name="separator"/>
   <addaction name="actionMonitor"/>
   <addaction name="actionMessages"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionTest"/>
   <addaction name="actionTestSettings"/>
   <addaction name="actionTestScreen"/>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QDockWidget" name="dwContent">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="wContent">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="tvContent"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwMessenger">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="wMessage">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="teMessage"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwMonitor">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="wMonitor">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QGridLayout" name="wMonitorGrid" rowstretch="0" columnstretch="0" rowminimumheight="0"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="cut" resource="qthpconnect.qrc">
     <normaloff>:/icons/cut_32x32.png</normaloff>:/icons/cut_32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="action_copy" resource="qthpconnect.qrc">
     <normaloff>:/icons/copy_32x32.png</normaloff>:/icons/copy_32x32.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/paste_32x32.png</normaloff>:/icons/paste_32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>C&amp;lose all</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Ca&amp;scade</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>&amp;Next</string>
   </property>
  </action>
  <action name="actionMessages">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/chat_32x32.png</normaloff>:/icons/chat_32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Messages</string>
   </property>
  </action>
  <action name="actionMonitor">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/monitor_32x32.png</normaloff>:/icons/monitor_32x32.png</iconset>
   </property>
   <property name="text">
    <string>M&amp;onitor</string>
   </property>
  </action>
  <action name="actionCalculators">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/calc_tree_32x32.png</normaloff>:/icons/calc_tree_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Calc&amp;ulators</string>
   </property>
  </action>
  <action name="actionAbout_HP_Connect">
   <property name="icon">
    <iconset theme="about" resource="qthpconnect.qrc">
     <normaloff>:/icons/about_32x32.png</normaloff>:/icons/about_32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About HP Connect</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/preferences_32x32.png</normaloff>:/icons/preferences_32x32.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;references</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>&amp;Previous</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/content_32x32.png</normaloff>:/icons/content_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Cont&amp;ent</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/save_32x32.png</normaloff>:/icons/save_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/save_all_32x32.png</normaloff>:/icons/save_all_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionCreateNew">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/add_new_32x32.png</normaloff>:/icons/add_new_32x32.png</iconset>
   </property>
   <property name="text">
    <string>CreateNew</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/delete_32x32.png</normaloff>:/icons/delete_32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/add_icon_32x32.png</normaloff>:/icons/add_icon_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/note_32x32.png</normaloff>:/icons/note_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Lo&amp;g</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/clone_to_32x32.png</normaloff>:/icons/clone_to_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="toolTip">
    <string>Test Button</string>
   </property>
  </action>
  <action name="actionTreePreference">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/preferences_32x32.png</normaloff>:/icons/preferences_32x32.png</iconset>
   </property>
   <property name="text">
    <string>TreePreference</string>
   </property>
  </action>
  <action name="actionTestSettings">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/preferences_32x32.png</normaloff>:/icons/preferences_32x32.png</iconset>
   </property>
   <property name="text">
    <string>TestSettings</string>
   </property>
  </action>
  <action name="actionTestScreen">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/add_background_32x32.png</normaloff>:/icons/add_background_32x32.png</iconset>
   </property>
   <property name="text">
    <string>TestScreen</string>
   </property>
   <property name="toolTip">
    <string>Test get Screen Function</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/refresh_32x32.png</normaloff>:/icons/refresh_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="qthpconnect.qrc">
     <normaloff>:/icons/preferences_32x32.png</normaloff>:/icons/preferences_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qthpconnect.qrc"/>
 </resources>
 <connections/>
</ui>
